### ISSUE TITLE
Accessibility: Add to cart on single product not announced to screen readers

### ISSUE THREAD

**Issue by amberhinds** [woothemes/woocommerce#43614]
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

When adding to the cart on a product single, the page reloads, and a banner notice is visible:
<img width="1106" alt="Screenshot 2024-01-14 at 5 07 44 PM" src="https://github.com/woocommerce/woocommerce/assets/13153456/d7ad478b-abc5-4aa2-aac1-69dc88a5bfb9">

This banner is not announced to screen reader users, and unless they explore the page, they will not know if their add-to-cart was successful.

Because this message appears above the H1, if they are using heading shortcuts to navigate the page, they are likely to miss the banner.

### Expected behavior

When a product is added to the cart, there should be an auditory message for screen reader users to know that the add-to-cart action was successful.

You can either do this by:
A) When the page reloads, set keyboard focus on the status message so it is read out. OR
B) Add an [aria-live](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-live) region on the status message and trigger it after the page loads.

### Actual behavior

Screen reader users don't easily know when a product is added to the cart.

### Steps to reproduce

1. Turn on a screen reader such as VoiceOver or NVDA.
2. Go to the single product page.
3. Add a product to your cart.
4. Listen as the page reloads.

### WordPress Environment

N/A

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woo.com/products/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

**Comment by dinhtungdu** [comment]
Thanks! I can confirm the current behavior and added this to our backlog.

**Comment by Manussakis** [comment]
I have started working on this issue.
